src/utils/source.js

```diff
@@ -462,5 +462,6 @@ export function isGenerated(source: Source) {
 }
 
 export function getSourceQueryString(source: ?Source) {
-  return source ? parseURL(source.url).search : "";
+  const query = source ? parseURL(source.url).search : "";
+  return query.replace(/:formatted$/, "");
```

Comment:
We have a function that does this replace; `getRawSourceURL`; if you update to that we can get this in.  Thank you!

Reply:
Thanks!